### PR TITLE
Engineering Supply Crate Additions

### PR DESCRIPTION
**Pull request by Momoberry**
[QOL]
## About the PR 
Adds three new supply pack options from QM. The major one is a replacement Disposal Pipe Cart, as someone stealing this early destroys the entire piping economy for engineering on several maps. The two minors ones are the portable fuel tank, as I know this has been something people have wanted for a while in QM, and a 2 piece set of a scrubber and air pump for atmos filtering.

## Why's this needed? 
I've had quite a few rounds where I just am unable to fix pipe issues due to a staffie taking the pipe cart for their personal gimmick and then tossing it in a dark corner, never to be seen again. I know the devs want people to try and fix piping issues, so this is hopefully a push in the right direction so people aren't left without the tools for it. 

The portable fuel tank is just for convenience for people, portable fuel tanks are a hot commodity, and with the "semi-recent" welder fuel changes, an engineer/QM having one of these on hand is pretty nice. That and crime. People like crime. The filtration pack is more of a direct way to get a scrubber and an air pump, without having a ton of left over gear that nobody wants and just looks ugly on the ground. The "atmospherish" pack has a scrubber, but it also has several tanks of gases, and a gimmick jumpsuit for 8,000. We have the emergency pack for the tanks of gases, so ideally i'd like to just remove it, but i don't wanna step on any toes. I think the meme of it is kind of discourages people.

tl;dr... I don't like map dependant gear, and I'd like having options to get them without the map spawns for generic gear. 🐝 

## Changelog

```changelog
(u)MomoBerry
(+)Removed Atmospherish Crate from Cargo. Added crates for pipe dispenser, portable fuel tanks, and a two piece scrubber/airtank set.
```

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
Air pumps should be added to the atmospherics supplies crate instead of a whole other crate in its entirety for one more item, god knows cargo has enough redundant crates as is (3 crates just for glowsticks)

**Comment by Momoberry** [comment]
I was tempted, but a lot of people dislike the bloat of items IN crates themselves, and usually only take a single item away from a crate of 5 different items, etc. I'd prefer to have obvious packs that have things that are very closely related, instead of throwing a bunch of somewhat related items together. I honestly might just delete atmospherish, i think it's outdated. I'll be going through cargo a bunch in the future and trimming down a lot of the bloat that you're talking about, but I think the main cause of the bloat is a bad UI. 

Other servers have many more items with a much less bloated menu than us. It's just an old menu, no submenus, etc.

File: code/modules/economy/supply_packs.dm [hunk 1]
```diff
@@ -294,6 +294,24 @@ ABSTRACT_TYPE(/datum/supply_packs)
...
+/datum/supply_packs/Disposal_Pipe_Cart
+	name = "Disposal Pipe Dispenser Cart"
+	desc = "Has a pesky staff assistant stolen your cart?"
+	category = "Engineering Department"
+	contains = list(/obj/machinery/disposal_pipedispenser/mobile)
+	cost = 4000
+	containertype = /obj/storage/crate
+	containername = "Replacement Disposal Cart Crate"
+
+/datum/supply_packs/Gas_Filtration
```
Comment (flrsh) [inline code/modules/economy/supply_packs.dm:306]:
```suggestion
/datum/supply_packs/gas_filtration
```

File: code/modules/economy/supply_packs.dm [hunk 2]
```diff
@@ -294,6 +294,24 @@ ABSTRACT_TYPE(/datum/supply_packs)
 	containertype = /obj/storage/crate
 	containername = "Station Pressurization Crate"
 
+/datum/supply_packs/Disposal_Pipe_Cart
```
Comment (flrsh) [inline code/modules/economy/supply_packs.dm:297]:
```suggestion
/datum/supply_packs/disposal_pipe_cart
```

**Review by flrsh — COMMENTED** [review]
Could you edit your changelog to past tense and list the new supply packs?

**Comment by Momoberry** [comment]
> Could you edit your changelog to past tense and list the new supply packs?

All good to go!

File: code/modules/economy/supply_packs.dm [hunk 3]
```diff
@@ -621,6 +639,15 @@ ABSTRACT_TYPE(/datum/supply_packs)
 	containertype = /obj/storage/crate
 	containername = "Assorted Glowsticks Crate - 4 pack"
 
+/datum/supply_packs/portable_fueltank
+	name = "Portable Welding Fuel Tank"
+	desc = "A single transportable fuel tank, for when you're on the move."
+	category = "Basic Materials"
+	contains = list(/obj/item/reagent_containers/food/drinks/fueltank )
```
Comment (frawhst) [inline code/modules/economy/supply_packs.dm:646]:
```suggestion
	contains = list(/obj/item/reagent_containers/food/drinks/fueltank)
```
the smallest of gripes

**Review by frawhst — APPROVED** [review]

File: code/modules/economy/supply_packs.dm (hunk 3 above)
Comment (Momoberry) [reply code/modules/economy/supply_packs.dm:646]: Consistency is important! One sec, will fix!
